### PR TITLE
Updates some food and cooking code to modern style standards

### PR DESCRIPTION
**Pull request by XyzzyThePretender**
[cleanliness]
## About the PR 
Regexed some booleans to TRUE/FALSE.
Changed for loops to 'for-in-to' and 'as anything' style as needed.
Cleaned unusual spacing.
Standardized item var to 'I' (from P or S).
Deleted unnecessary 'as mob' from procs.
Also found a stray space in one of the maps.

## Why's this needed? 
Needed to look into the cooking code for a potential mycology/yeast implementation, so I cleaned a bit while I'm here.
Regex replacement went all over the place: the main cleanup was in cooking.dm, cookingrecipes.dm, and food_and_drink.dm.
Standardization good.

### CODE REVIEW

File: code/obj/submachine/cooking.dm [hunk 1]
```diff
@@ -824,206 +829,208 @@ table#cooktime a#start {
...
 		src.icon_state = "processor-on"
-		src.working = 1
+		src.working = TRUE
 		src.visible_message("The [src] begins processing its contents.")
-		sleep(rand(30,70))
+		sleep(rand(3, 7) SECONDS)
 		// Dispense processed stuff
-		for(var/obj/item/P in src.contents)
-			switch( P.type )
+		for(var/obj/item/I as anything in src.contents)
```
Comment (ZeWaka) [inline code/obj/submachine/cooking.dm:873]: no, not everything inside people is an item

File: code/obj/submachine/cooking.dm [hunk 2]
```diff
@@ -1265,17 +1273,17 @@ var/list/mixer_recipes = list()
 							F.unlock_medal_when_eaten = "That tasted funny"
 						else
 							F.unlock_medal_when_eaten = "Space Ham" //replace the old fat person method
-				for (var/obj/item/I in to_remove)
+				for (var/obj/item/I as anything in to_remove)
 					qdel(I)
 				to_remove.len = 0
 
-			for (var/obj/I in src.contents)
+			for (var/obj/I as anything in src.contents)
```
Comment (ZeWaka) [inline code/obj/submachine/cooking.dm:1280]: same here and elsewhere

File: code/obj/submachine/cooking.dm [hunk 3]
```diff
@@ -3,41 +3,41 @@
...
 		else if (istype(W, /obj/item/mop)) // dude whatever
 			var/fill = W.reagents.maximum_volume
 			if (W.reagents.total_volume >= fill)
-				user.show_text("[W] is too wet already.", "red")
+				boutput(user, "<span class='alert'>[W] is too wet already.</span>")
 			else
 				fill -= W.reagents.total_volume
 				W.reagents.add_reagent("water", fill)
-				user.show_text("You wet [W].", "blue")
+				boutput("<span class='notice'>You wet [W].</span>")
```
Comment (ZeWaka) [inline code/obj/submachine/cooking.dm:40]:
```suggestion
				boutput(user, "<span class='notice'>You wet [W].</span>")
```

File: code/obj/submachine/cooking.dm [hunk 4]
```diff
@@ -3,41 +3,41 @@
...
 		else if (istype(W, /obj/item/reagent_containers/glass/) || istype(W, /obj/item/reagent_containers/food/drinks/) || istype(W, /obj/item/reagent_containers/balloon/) || istype(W, /obj/item/soup_pot))
 			var/fill = W.reagents.maximum_volume
 			if (W.reagents.total_volume >= fill)
-				user.show_text("[W] is too full already.", "red")
+				boutput(user, "<span class='alert'>[W] is too full already.</span>")
 			else
 				fill -= W.reagents.total_volume
 				W.reagents.add_reagent("water", fill)
-				user.show_text("You fill [W] with water.", "blue")
+				boutput("<span class='notice'>You fill [W] with water.</span>")
```
Comment (ZeWaka) [inline code/obj/submachine/cooking.dm:31]:
```suggestion
				boutput(user, "<span class='notice'>You fill [W] with water.</span>")
```

File: code/obj/submachine/cooking.dm [hunk 5]
```diff
@@ -81,23 +81,24 @@
...
+	var/doing_a_thing = FALSE
 
 	attack_hand(var/mob/user)
 		src.add_dialog(user)
 		var/dat = "<b>Ice Cream-O-Mat 9900</b><br>"
 		if(src.cone)
 			dat += "<a href='?src=\ref[src];eject=cone'>Eject Cone</a><br>"
 			dat += "<b>Select a Flavor:</b><br><ul>"
-			for(var/flavor in flavors)
+			for(var/flavor as anything in flavors)
```
Comment (ZeWaka) [inline code/obj/submachine/cooking.dm:101]: unnecessary 
```suggestion
			for(var/flavor in flavors)
```

File: code/obj/submachine/cooking.dm [hunk 6]
```diff
@@ -318,17 +321,17 @@ table#cooktime a#start {
...
-			<b>Time:</b> [time]<br>
-			<b>Heat:</b> [heat]<br>
+			<b>Time:</b> [src.time]<br>
+			<b>Heat:</b> [src.heat]<br>
 			<hr>
 		"}
 		if (!src.working)
 			var/junk = ""
-			for (var/obj/item/I in src.contents)
+			for (var/obj/item/I as anything in src.contents)
```
Comment (ZeWaka) [inline code/obj/submachine/cooking.dm:330]: x

File: code/obj/submachine/cooking.dm [hunk 7]
```diff
@@ -600,38 +603,41 @@ table#cooktime a#start {
...
+			if (src.heat == "High") src.time *= 2
 
 			// If emagged produce random output.
-			if (emagged)
+			if (src.emagged)
 				// Enforce GIGO and prevent infinite reuse
-				var/contentsok = 1
-				for(var/obj/item/I in src.contents)
+				var/contentsok = TRUE
+				for(var/obj/item/I as anything in src.contents)
```
Comment (ZeWaka) [inline code/obj/submachine/cooking.dm:620]: x

**Review by ZeWaka — CHANGES_REQUESTED** [review]

File: code/obj/submachine/cooking.dm [hunk 8]
```diff
@@ -1265,17 +1273,17 @@ var/list/mixer_recipes = list()
 							F.unlock_medal_when_eaten = "That tasted funny"
 						else
 							F.unlock_medal_when_eaten = "Space Ham" //replace the old fat person method
-				for (var/obj/item/I in to_remove)
+				for (var/obj/item/I as anything in to_remove)
```
Comment (XyzzyThePretender) [inline code/obj/submachine/cooking.dm:1276]: type is set by line 1219

**Comment by ZeWaka** [comment]
?

**Pull request closed without merging**